model-fcrepo/src/test/java/edu/unc/lib/boxc/model/fcrepo/objects/FolderObjectTest.java

```diff
@@ -116,13 +116,13 @@ public void addFolderTest() {
     @Test
     public void addWorkTest() {
         WorkObjectImpl childObj = new WorkObjectImpl(childPid, driver, repoObjFactory);
-        when(repoObjFactory.createWorkObject(any(Model.class))).thenReturn(childObj);
+        when(repoObjFactory.createWorkObject(isNull())).thenReturn(childObj);
 
         WorkObject workObj = folder.addWork();
 
         verify(repoObjFactory).createWorkObject(null);
 
-        assertTrue("Incorrect type of child added", workObj instanceof WorkObject);
+        assertTrue("Incorrect type of child added", workObj != null);
```

Comment:
Why did this need to change? The factory should still be returning the WorkObjectImpl initialized above.

Reply:
I think it was suggested by Intellij. I'm fine with reverting it.